web/templates/compose_banner/compose_banner.hbs

```diff
@@ -12,6 +12,9 @@
         {{#if button_text}}
         <button class="main-view-banner-action-button{{#if hide_close_button}} right_edge{{/if}}" {{#if scheduling_message}}data-validation-trigger="schedule"{{/if}}>{{button_text}}</button>
         {{/if}}
+        {{#if is_onboarding_banner}}
+        <button class="main-view-banner-action-button right_edge" data-action="mark-as-read">{{t "Got it"}}</button>
+        {{/if}}
```

Comment:
What causes clicking this button to mark the banner as read?

Reply:
Ahh, OK, it is the `data-action` hook, and `change setting` doesn't do that, makes sense.